browser-api/src/main/java/com/duckduckgo/app/notification/model/SchedulableNotification.kt

```diff
@@ -26,8 +26,6 @@ import androidx.annotation.StringRes
  */
 interface SchedulableNotification {
     val id: String
-    val launchIntent: String
-    val cancelIntent: String
```

Comment:
Removed the `launchIntent`, `cancelIntent` from here as they are now in the notification plugin.